### PR TITLE
Fix typo with RSS typed subscription queries

### PR DESCRIPTION
**Pull request by jacksonh**

### CODE REVIEW

**Comment by jacksonh** [comment]
cc @sywhb

File: packages/api/src/resolvers/subscriptions/index.ts [hunk 1]
```diff
@@ -83,7 +83,7 @@ export const subscriptionsResolver = authorized<
         status: SubscriptionStatus.Active,
       })
     } else if (type && type == SubscriptionType.Rss) {
-      queryBuilder.where({
+      queryBuilder.andWhere({
```
Comment (sywhb) [inline packages/api/src/resolvers/subscriptions/index.ts:86]: 👍

**Review by sywhb — APPROVED** [review]
LGTM

**Comment by jacksonh** [comment]
Hopefully the end of this long saga. Maybe I can not write backend code anymore.

**Comment by sywhb** [comment]
> Hopefully the end of this long saga. Maybe I can not write backend code anymore. 

The current setup makes it still quite troublesome to write tests...

**Pull request merged**